lib/CMakeLists.txt

```diff
@@ -117,6 +117,9 @@ if (WIN32)
         "src/facts/windows/collection.cc"
         "src/util/windows/wsa.cc"
     )
+    if (OPENSSL_FOUND)
```

Comment:
this is now duplicate with condition at line 103, could this be executed once outside of UNIX/WIN32 checks?

Reply:
Agreed and done.